src/common/libsubprocess/test/subprocess.c

```diff
@@ -325,7 +325,7 @@ void output_cb (flux_subprocess_t *p, const char *stream)
         sprintf (cmpbuf, "%s:hi\n", stream);
 
         ok (!strcmp (ptr, cmpbuf),
-            "flux_subprocess_read_line returned correct data", stream);
```

Comment:
Since this commit only modifies tests, it should probably have a testsuite: prefix not libsubprocess:

Reply:
yup, just realized it should have been `libsubprocess/test`.